src/applications/check-in/components/WhatToDoNext.jsx

```diff
@@ -81,27 +58,49 @@ const WhatToDoNext = props => {
               const cardTitleId = `what-next-card-title-${index}`;
               let cardTitle = t(
                 'its-time-to-check-in-for-your-time-appointment',
-                {
-                  time: new Date(appointment.startTime),
-                },
               );
               if (app === APP_NAMES.PRE_CHECK_IN) {
-                cardTitle = getPreCheckinCardTitle();
+                cardTitle = t(
+                  'review-your-contact-information-for-your-appointment',
+                  { count: actionableAppointments.length },
+                );
               }
               return (
                 <div
                   className="vads-u-margin-bottom--2"
                   key={appointment.appointmentIen}
                   data-testid="what-next-card"
                 >
-                  <va-card show-shadow={actionableAppointments.length > 1}>
+                  <va-card show-shadow={appointmentCards.length > 1}>
                     <h3
                       className="vads-u-margin-top--0 vads-u-font-size--h4"
                       data-testid="what-next-card-title"
                       id={cardTitleId}
                     >
                       {cardTitle}
                     </h3>
+                    <ul>
```

Comment:
We discussed showing just a single line without all the list markup here when there is only one item. Is that going to be done in a separate ticket?

Reply:
there is a ticket for it: https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/89076

Also has a planning poker out for it.